test/packages/good/manifest.yml

```diff
@@ -4,8 +4,11 @@ title: Good package
 description: This package is good.
 version: 1.0.0
 type: integration
+source:
+  license: "Apache-2.0"
```

Comment:
Umm, I consider this property implements this comment. This is a value, from an enum of allowed values. How does it violate the comment?

Reply:
How would the `licensing` structure be? Something like this?
```
licensing:
  license: 'Apache-2.0'
  subscription: 'basic'
```

Reply:
In case we have multiple licenses per package we could use that structure:

```
licensing:
   licenses: [
      name: Apache-2.0,
      path: LICENSE.txt
   ]
   subscription: basic
```

Sometimes there is also `NOTICE.txt` file, which seems to be related to package license.